docs/devcontainers/create-dev-container.md

```diff
@@ -254,14 +254,12 @@ VS Code can be configured to **automatically start any needed containers** for a
 
 To get started quickly, **open the folder** you want to work with in VS Code and run the **Dev Containers: Add Dev Container Configuration Files...** command in the Command Palette (`kbstyle(F1)`).
 
-![Select Docker Compose File](images/create-dev-container/select-docker-compose.png)
-
-You'll be asked to either select an existing Docker Compose file (if one exists), or pick a pre-defined container configuration from the [vscode-dev-containers repository](https://github.com/microsoft/vscode-dev-containers) in a filterable list sorted based on your folder's contents. Many of these "dev container definitions" use a Dockerfile, so select one of the following definitions as a starting point for Docker Compose:
+You'll be prompted to pick a pre-defined container configuration from our [first-party and community index](https://containers.dev/templates) in a filterable list sorted based on your folder's contents. From the VS Code UI, you may select one of the following Templates as a starting point for Docker Compose:
 
-* [Existing Docker Compose](https://aka.ms/vscode-remote/samples/existing-docker-compose)
-* [Node.js & MongoDB](https://aka.ms/vscode-remote/samples/node-mongo)
-* [Python & PostgreSQL](https://aka.ms/vscode-remote/samples/python-postgres)
-* [Docker-from-Docker Compose](https://aka.ms/vscode-remote/samples/docker-from-docker-compose).
+* [Existing Docker Compose](https://github.com/devcontainers/templates/tree/main/src/docker-existing-docker-compose) - Includes a set of files that you can drop into an existing project that will reuse a `docker-compose.yml` file in the root of your project.
+* [Node.js & MongoDB](https://github.com/devcontainers/templates/tree/main/src/javascript-node-mongo) -  A Node.js container that connects to a Mongo DB in a different container.
```

Comment:
"to a MongoDB database in a..."

Reply:
Might want to update the template README as well. 
"Mongo DB" -> "MongoDB"
"a Mongo DB" -> "a MongoDB database"

Reply:
Thanks, updated in this doc and updated a PR for the Template: https://github.com/devcontainers/templates/pull/46.